app/models/Ticket.scala

```diff
@@ -0,0 +1,21 @@
+package models
+
+case class Ticket(rowNumber: Int, seatNumberInRow: Int, ticketType: TicketType) {
+  override def toString: String = s"Row number: $rowNumber <--> seat number: ${seatNumberInRow + 1} <--> ticket type: $ticketType <--> ticket's price: ${ticketType.price} zł"
+}
+
+abstract class TicketType {
+  val price: Double
+}
+
+class ChildTicket(override val price: Double = 12.5) extends TicketType {
```

Comment:
in real/production app, where you'd store the ticket prices?

Reply:
Like I mentioned in the first comment above - in a database inside a table:
TicketType(Id: Int Primary Key, name: String, price: Decimal)
I understand - hardcoding prices inside the code is really not a good idea. Prices must be independent from the code and easy to change and set without changing the program.